python/ray/util/sgd/v2/backends/backend.py

```diff
@@ -319,6 +358,29 @@ def shutdown(self):
         self.worker_group.shutdown()
         self.worker_group = InactiveWorkerGroup()
 
+    @property
+    def logdir(self) -> str:
+        """Path to the log directory."""
```

Comment:
I'm wondering if it might be easier to just set the logdir once during `__init__`, and then have this just return `self._logdir`.

Reply:
Yeah this makes sense. Updated and removed the `@property` altogether so it's just a direct attribute.